hack/helm/release/cmd/root.go

```diff
@@ -0,0 +1,171 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/elastic/cloud-on-k8s/hack/helm/release/internal/helm"
+)
+
+const (
+	// viper flags
+	chartsDirFlag       = "charts-dir"
+	chartsRepoURLFlag   = "charts-repo-url"
+	credentialsFileFlag = "credentials-file"
+	dryRunFlag          = "dry-run"
+	envFlag             = "env"
+	excludesFlag        = "excludes"
+	enableVaultFlag     = "enable-vault"
+	vaultSecretFlag     = "vault-secret"
+
+	// GCS Helm Buckets
+	elasticHelmChartsDevBucket  = "elastic-helm-charts-dev"
+	elasticHelmChartsProdBucket = "elastic-helm-charts"
+
+	// Helm Repositories
+	elasticHelmChartsDevRepoURL  = "https://helm-dev.elastic.co/helm"
+	elasticHelmChartsProdRepoURL = "https://helm.elastic.co/helm"
+
+	// Environment flag options
+	devEnvironment  = "dev"
+	prodEnvironment = "prod"
+)
+
+var (
+	bucket        string
+	chartsRepoURL string
+)
+
+func init() {
+	cobra.OnInitialize(initConfig)
+}
+
+func releaseCmd() *cobra.Command {
+	releaseCommand := &cobra.Command{
+		Use:     "release",
+		Short:   "Release ECK Helm Charts",
+		Long:    `This command will Release ECK Helm Charts to a given Environment.`,
+		Example: fmt.Sprintf("  %s", "release --charts-dir=./deploy --upload-index --dry-run=false"),
```

Comment:
I think `--upload-index` has been removed.

```suggestion
		Example: fmt.Sprintf("  %s", "release --charts-dir=deploy --dry-run=false"),
```